deploy/helm/kubecf/templates/uaa_autoscaler_client.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.features.autoscaler.enabled -}}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Name }}.var-cf-client-id
+  namespace: {{ .Release.Namespace | quote }}
+type: Opaque
+stringData:
+  value: app_autoscaler
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ops-uaa-autoscaler-client
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/name: {{ include "kubecf.fullname" . }}
+    app.kubernetes.io/version: {{ default .Chart.Version .Chart.AppVersion | quote }}
+    helm.sh/chart: {{ include "kubecf.chart" . }}
+data:
+  ops: |
+    - type: replace
```

Comment:
Not sure why this doesn't live in the main autoscaler ops file, but… okay?

Reply:
This should live in the main autoscaler ops-file.

Reply:
Removed this file and add it to the main autoscaler ops file.